files/image_config/logrotate/logrotate.d/rsyslog

```diff
@@ -85,7 +85,22 @@
     endscript
     postrotate
         if [ $(echo $1 | grep -c "/var/log/swss/") -gt 0 ]; then
-            pgrep -x orchagent | xargs /bin/kill -HUP 2>/dev/null || true
+            # for multi asic platforms, there are multiple orchagents
+            # send the SIGHUP only to the orchagent the which needs log file rotation
+            PLATFORM=`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`
+            ASIC_CONF=/usr/share/sonic/device/$PLATFORM/asic.conf
+            if [ -f "$ASIC_CONF" ]; then
+                . $ASIC_CONF
+            fi
+            if [ $NUM_ASIC -gt 1 ]; then
+                log_file=$1
+                log_file_name=${log_file#/var/log/swss/}
+                logger -p syslog.info -t "logrotate" "Sending SIGHUP to OA log_file_name:$log_file_name"
+                pgrep -xa orchagent | grep $log_file_name | awk '{ print $1; }' | xargs /bin/kill -HUP 2>/dev/null || true
+            else
+                logger -p syslog.info -t "logrotate" "Sending SIGHUP to OA"
```

Comment:
one minor comment i think for consistency if we can keep same syslog message format for both single and multi-asic.
Display file name for both.